tests/conftest.py

```diff
@@ -47,3 +48,7 @@ def _initialize_test_directory(session):
 
 def pytest_sessionstart(session):
     _initialize_test_directory(session)
+
+
+def pytest_assertrepr_compare(op, left, right):
```

Comment:
custom reporting hook for pytest
https://docs.pytest.org/en/latest/assert.html#defining-your-own-assertion-comparison